dom/apps/AppsUtils.jsm

```diff
@@ -956,7 +956,7 @@ ManifestHelper.prototype = {
       if (this._localeProp("start_url")) {
         return this._baseURI.resolve(this._localeProp("start_url") || "/");
       }
-      return this._baseURI.resolve(this._localeProp("launch_path") || "/");
```

Comment:
I think we cannot do that:
- It might break other hosted apps that relies on `/`. 
- Anyway, this launch_path field is required. Apps that don't have it won't even install AFAIK.

What are you trying to fix exactly? Is it the "some apps like Firesea IRC do not start" bug?